indexing-hadoop/pom.xml

```diff
@@ -67,6 +67,10 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-aws</artifactId>
```

Comment:
this is aws hadoop, which many people do not run on, and will probably not work with stock hadoop. I think a proper way to support multiple hadoop versions is worth a discussion in the forums